src/tiling/tile_state.rs

```diff
@@ -144,4 +168,4 @@ impl<'a, T: Pixel> TileStateMut<'a, T> {
       y: self.sbo.y + tile_sbo.y,
     }
   }
-}
+}
```

Comment:
Please configure your editor or run rustfmt before pushing.

Reply:
I manually check for these kinds of things right now. I will run rustfmt from now on.